lib/src/VirtualObjectIdManager.cpp

```diff
@@ -242,7 +243,7 @@ sai_object_id_t VirtualObjectIdManager::allocateNewObjectId(
 
     if (objectIndex > SAI_REDIS_OBJECT_INDEX_MAX)
     {
-        SWSS_LOG_THROW("no more object indexes available, given: 0x%lx but limit is 0x%llx",
+        SWSS_LOG_THROW("no more object indexes available, given: 0x%" PRIu64 " but limit is 0x%" PRIu64 " ",
```

Comment:
PRIx64 ?

Reply:
My thinking is that PRIu64 is the way to go here given the following

    uint64_t objectIndex = m_oidIndexGenerator->increment();

and ...

    #define SAI_REDIS_OBJECT_INDEX_MAX ( (1ULL << SAI_REDIS_OBJECT_INDEX_BITS_SIZE) - 1 )
    #define SAI_REDIS_OBJECT_INDEX_MASK (SAI_REDIS_OBJECT_INDEX_MAX)

Both cases indicate unsigned.

Reply:
ok